src/hooks/users/useWorldUsers.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { createContext, useContext, useMemo } from "react";
+import { isEqual } from "lodash";
+
+import { User } from "types/User";
+
+import { WithId } from "utils/id";
+import { worldUsersWithoutLocationSelector } from "utils/selectors";
+import { isDefined } from "utils/types";
+
+import { isLoaded, useFirestoreConnect } from "hooks/useFirestoreConnect";
+import { useSelector } from "hooks/useSelector";
+import { useSovereignVenueId } from "hooks/useSovereignVenueId";
+
+const noUsers: WithId<User>[] = [];
+
+export interface WorldUsersContextState {
+  isSovereignVenueIdLoading: boolean;
+  sovereignVenueId?: string;
+  sovereignVenueIdError?: string;
+
+  shouldConnect: boolean;
+}
+
+const WorldUsersContext = createContext<WorldUsersContextState | undefined>(
+  undefined
+);
+
+export interface WorldUsersProviderProps {
+  venueId?: string;
+}
+
+export const WorldUsersProvider: React.FC<WorldUsersProviderProps> = ({
+  venueId,
+  children,
+}) => {
+  const {
+    sovereignVenueId,
+    isSovereignVenueIdLoading,
+    errorMsg: sovereignVenueIdError,
+  } = useSovereignVenueId({
+    venueId,
+  });
```

Comment:
probably has conflicts with the recent sovereignVenue changes

Reply:
Yeah, makes sense. Though should also be a fairly trivial merge to get it fixed up.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1703/commits/ef36347c3d89ff3877cb51ee7d81e5a175b22040